File: lib/app/Models/CreateChatRequest.php
<?php
/**
 * CreateChatRequest
 */
namespace app\Models;

/**
 * CreateChatRequest
 * @description Заказ, для которого нужно создать чат.
 */
class CreateChatRequest {

    /** @var int $orderId Идентификатор заказа на Маркете.*/
    public $orderId = 0;

}

File: lib/Models/CreateChatRequest.cpp


#include "CreateChatRequest.h"

using namespace Tiny;

CreateChatRequest::CreateChatRequest()
{
	orderId = long(0);
}

CreateChatRequest::CreateChatRequest(std::string jsonString)
{
	this->fromJson(jsonString);
}

CreateChatRequest::~CreateChatRequest()
{

}

void
CreateChatRequest::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);

    const char *orderIdKey = "orderId";

    if(object.has_key(orderIdKey))
    {
        bourne::json value = object[orderIdKey];



        jsonToValue(&orderId, value, "long");


    }


}

bourne::json
CreateChatRequest::toJson()
{
    bourne::json object = bourne::json::object();





    object["orderId"] = getOrderId();



    return object;

}

long
CreateChatRequest::getOrderId()
{
	return orderId;
}

void
CreateChatRequest::setOrderId(long  orderId)
{
	this->orderId = orderId;
}



